### DISCUSSION TITLE
What is the path `/ws`?

### DISCUSSION THREAD

**Discussion by BackedUpBooty** [souramoo/commentoplusplus#107] [Q&A]
This isn't a commento problem, but the answer would help me out. My commento domain is being requested multiple times a second from multiple locations with the path `commento.domain.com/ws`. As far as I can tell that's not a valid path, but I wanted to see if anybody else had experienced something similar, and if it was ok to just completely block access to it?

**Comment by souramoo** [comment]
This is for the websocket connection to provide real time comment updates without refreshing the page.

You can disable it with a setting if you like, or switch to polling mode (slower and less efficient bandwidth wise)

**Reply by BackedUpBooty** [reply]
Ah understood. Is there any reason you can think of that Cloudflare would be is seeing around 5,000 attempts to access it an hour? My actual site doesn't even get that!

Is there any security issue surrounding it? Provided I don't need to worry about it then I don't have an issue.

To disable it can I just add `data-no-websockets="true"` to the `envirnoment:` part of my docker-compose?

**Reply by souramoo** [reply]
You can disable it by adding that attribute to the tag where you include commento rather than the docker file! No security issues involved, just a convenience feature.

Normally the reason for getting a lot of requests like that is because commento is running behind a proxy that doesn't forward websocket connections (or is not configured to do so!) but isn't told not to use it and so keeps trying to connect!

**Reply by BackedUpBooty** [reply]
So the nginx configuration in swag is a little different to the documentation you provided for websocket (e.g. in my proxy.conf I have `proxy_set_header Connection $connection_upgrade;` but the commento docs say to use `proxy_set_header Connection "Upgrade";`) but I'll try the ones in the documentation and see if that makes a difference
